charts/pega/charts/installer/config/prlog4j2.xml

```diff
@@ -88,8 +88,18 @@
 				<SizeBasedTriggeringPolicy size="250 MB"/>
 			</Policies>
 			<DefaultRolloverStrategy max="20"/>
-		</RollingRandomAccessFile> 
-		
+		</RollingRandomAccessFile>
+
+		<!-- RollingFile Appender for PegaCLUSTER logs -->
+		<RollingRandomAccessFile  name="CLUSTER" fileName="${sys:pega.logdir}/PegaCLUSTER.log" filePattern="${sys:pega.logdir}/PegaCLUSTER-%d{MM-dd-yyyy}-%i.log.gz">
```

Comment:
fileName and filePattern values might need to computed in other way with installer. Check other appender configuration [here](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/charts/installer/config/prlog4j2.xml#L13)